plugins/circleci/src/components/App.tsx

```diff
@@ -25,11 +25,8 @@ export const App = () => {
     <AppStateProvider>
       <>
         <Routes>
-          <Route path="/circleci" element={<BuildsPage />} />
-          <Route
-            path="/circleci/build/:buildId"
-            element={<DetailedViewPage />}
-          />
+          <Route path="*" element={<BuildsPage />} />
```

Comment:
@shmidt-i @Nek how come this plugin handles Routes like this instead of using the plugin API?

Reply:
This was a result of the discussion at that moment, `AppStateProvider` shared between routes being one of the main reasons behind. Maybe soon we should revisit this approach ;)